account_reversal/README.rst

```diff
@@ -1,23 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
 Account Reversal
 ================
 
 This module adds an action "Reversal" on account moves,
 to allow the accountant to create reversal account moves in 2 clicks.
 Also add on account entries:
 
- * a checkbox and filter "to be reversed"
- * a link between an entry and its reversal entry
+* a checkbox and filter "to be reversed"
+* a link between an entry and its reversal entry
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/9.0
+
+
+Credits
+=======
 
 Module originally developped by Alexis de Lattre <alexis.delattre@akretion.com>
```

Comment:
Remove this text, as it's not the template. He is already in the contributors part and the git history.